MANIFEST.in

```diff
@@ -1,5 +1,8 @@
 include pigar/.db.sqlite3
+include CHANGELOGS.rst
 include README.rst
+include README-PYPI.rst
```

Comment:
Also appears [these files are needed by `setup.py`]( https://github.com/damnever/pigar/blob/v0.7.2/setup.py#L20-L24 ) and they are not included in the `sdist` currently. Added them here so we can get that fixed as well. Should note this breaks the `pip install` of `sdist`s.